src/main/java/com/conveyal/datatools/manager/UpdateSQLFeedsMain.java

```diff
@@ -1,131 +1,107 @@
 package com.conveyal.datatools.manager;
 
+import com.conveyal.datatools.manager.utils.sql.NamespaceCheck;
+import com.conveyal.datatools.manager.utils.sql.SqlSchemaUpdater;
+import com.google.common.collect.Lists;
+
 import java.io.IOException;
 import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.sql.Statement;
 import java.util.ArrayList;
-import java.util.Collections;
+import java.util.Collection;
 import java.util.List;
 
 import static com.conveyal.datatools.manager.DataManager.initializeApplication;
 import static com.conveyal.datatools.manager.DataManager.registerRoutes;
 
 /**
- * Main method that performs batch SQL updates on optionally filtered set of namespaces over the GTFS SQL database
- * connection specified in the configuration files (env.yml).
- *
- * For example, this script can add a column for all `routes` tables for schemas that do not have a null value for
- * the filename field in the feeds table. In effect, this would alter only those feeds that are editor snapshots.
+ * Main method that checks for missing tables and columns
```

Comment:
Reworded comment in https://github.com/ibi-group/datatools-server/pull/459/commits/a7f11ec7a8272a68de1a19d3ac498f0ca0ca78f1. I place line breaks so that sentence parts that go together are on the same line.